mtgjson5/classes/mtgjson_sealed_product.py

```diff
@@ -1,13 +1,117 @@
 """
 MTGJSON Singular Sealed Product Object
 """
+import enum
 from typing import Any, Dict, Optional
 
 from ..utils import to_camel_case
 from .mtgjson_identifiers import MtgjsonIdentifiersObject
 from .mtgjson_purchase_urls import MtgjsonPurchaseUrlsObject
 
 
+class MtgjsonSealedProductCategory(enum.Enum):
```

Comment:
I'd prefer each of these enums to be defined in their own file, and then use the `__init__` to bring them together

Reply:
i think it makes sense to keep everything related to sealed in a single place no?